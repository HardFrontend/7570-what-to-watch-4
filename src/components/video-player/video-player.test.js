import React from "react";
import renderer from "react-test-renderer";
import VideoPlayer from "./video-player.jsx";

const film =
  {
    id: 0,
    title: `Fantastic Beasts`,
    img: `img/fantastic-beasts-the-crimes-of-grindelwald.jpg`,
    genre: `comedy`,
    year: `2011`,
    videosrc: `https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4`
  };


it(`VideoPlayer render`, () => {
  const tree = renderer
    .create(<VideoPlayer
      poster={film.img}
      src={film.videosrc}
      preload="none"
      loop={true}
      autoPlay={false}
    />, {
      createNodeMock: () => {
        return {};
      }
    }
    ).toJSON();

  expect(tree).toMatchSnapshot();
});
