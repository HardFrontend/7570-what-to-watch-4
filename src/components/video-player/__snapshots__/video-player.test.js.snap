// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VideoPlayer render 1`] = `
<video
  autoPlay={false}
  loop={true}
  poster="img/fantastic-beasts-the-crimes-of-grindelwald.jpg"
  preload="none"
  src="https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4"
/>
`;
